src/options/view.rs

```diff
@@ -266,6 +266,9 @@ impl TimeFormat {
         else if &word == "full-iso" {
             Ok(Self::FullISO)
         }
+        else if &word == "hu" {
```

Comment:
Why `hu` instead of `human` or `relative`?

Reply:
Yes! I also thought that after seeing nushell, exa should have come up with the idea of this PR, which was a temporary idea, so I didn't think much about it. However, now It's better to use "human" or "relative" instead.